src/launch.py

```diff
@@ -19,6 +21,8 @@ def typer_interface(config_file: str):
 
     @staticmethod
     def launch(config):
+
+        start_time = time.time()
```

Comment:
is this necessary? Are we logging this somewhere? If we are logging then let's use a .txt file